synapse/storage/database.py

```diff
@@ -701,15 +701,15 @@ def new_transaction(
                     exception_callbacks=exception_callbacks,
                 )
                 try:
-                    with opentracing.start_active_span(
+                    with tracing.start_active_span(
                         "db.txn",
```

Comment:
When viewing a trace for a servlet, we have the `db.{desc}` span but the child database spans are missing like `db.connection`, `db.txn`, `db.query` 🤔